### DISCUSSION TITLE
CLI: copy *.jpg *.avif

### DISCUSSION THREAD

**Discussion by ivanjaros** [libvips/libvips#4274] [General]
I have a bunch of sequentially named jpeg images produced by ffmpeg and I want to convert them all into avif format. ffmpeg is not being able to produce avif straight away so i have to do it afterwards, which is fine. Is there a wildcard pattern for input/output files that can handle this?

And i am not looking for a shell script. I want one simple cli command.

**Comment by jcupitt** [comment]
Hello again, no, libvips is single images only, though it does support animated and multipage images (webp, gif, avif, etc.).

FWIW, I would convert with:

```
parallel vips copy {.}.jpg {.}.avif ::: *.jpg
```

**Reply by ivanjaros** [reply]
hm, thought so. thanks.
